docs/conf.py

```diff
@@ -80,6 +80,10 @@ def __getattr__(cls, name):
     "xarray": ("http://xarray.pydata.org/en/stable/", None),
 }
 
+
+# Notebook execution (per cell)
+nb_execution_timeout = 120
```

Comment:
If Numba compilation times continue to grow we may want to get rid of bulky notebooks or use cache etc.

Reply:
The size of the notebook or dataset doesn't have an impact on the Numba compilation.